bundle/build.gradle

```diff
@@ -0,0 +1,97 @@
+/*
+ * ========================LICENSE_START=================================
+ * AEM Permission Management
+ * %%
+ * Copyright (C) 2013 Cognifide Limited
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+plugins {
+    id 'com.cognifide.aem.bundle'
+    id 'antlr'
+    id 'groovy'
+    id 'java'
+}
+
+description = 'AEM Permission Management :: Application Core'
+
+aem {
+    bundle {
+        attribute 'Bundle-Category', 'apm'
+        attribute 'Bundle-Vendor', 'Cognifide'
+        attribute 'Sling-Model-Packages', 'com.cognifide.cq.cqsm.core.models,com.cognifide.cq.cqsm.core.scripts,com.cognifide.cq.cqsm.api.history'
+        attribute 'Export-Package', 'com.cognifide.cq.cqsm.foundation.actions.*;com.cognifide.cq.cqsm.api.*;com.cognifide.cq.cqsm.core.models.*;com.cognifide.cq.cqsm.core.automaticexecutor.*'
+        attribute 'Import-Package', '!sun.misc,*'
+        attribute 'Sling-Nodetypes', 'CQ-INF/nodetypes/cqsm_nodetypes.cnd'
+        attribute 'CQ-Security-Management-Actions', 'com.cognifide.cq.cqsm.foundation.actions'
+    }
+}
+
+dependencies {
+    antlr "org.antlr:antlr4:4.7.1"
+
+    aemInstall group: 'com.cognifide.cq.actions', name: 'com.cognifide.cq.actions.api', version:'6.0.2'
```

Comment:
Just a suggestion: maybe could we store version in selected place like this:
`ext {
  versions = [
    cq_actions: "6.0.2",
    ...  ]
  libs = [
    ...
  ]
}`
I think it could improve readability ;)